platooning_strategic_IHP/include/platoon_strategic_ihp.h

```diff
@@ -532,13 +589,28 @@ namespace platoon_strategic_ihp
             * \return mobility operation msg.
             */
             cav_msgs::MobilityOperation composeMobilityOperationLeaderAborting();
-            
             /**
             * \brief Function to compose mobility operation in CandidateLeader.
             *
             * \return mobility operation msg.
             */
             cav_msgs::MobilityOperation composeMobilityOperationCandidateLeader();
+            /**
```

Comment:
In this whole section, please add a blank line after each method declaration (the compilable line) to make better visual grouping of declarations and their descriptions.

Reply:
no problem.